src/authentication.js

```diff
@@ -66,11 +68,11 @@ const launchPhone = async () => {
     if (session) {
       return onLogin(session);
     }
-  } catch(e) {
+  } catch (e) {
     displayAuthError(e);
   }
 
-  openLogin();
+  return openLogin();
```

Comment:
`openLogin` doesn't returns anything and is not async. There is not need to return the result of this function here.